velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.cpp

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
@majetideepak since you were the one creating this interface some time ago, could you help reviewing this PR?